documentation/admin.py

```diff
@@ -95,6 +99,10 @@ class BlockAdmin(PageAdmin, VersionAdmin):
         models.CharField: {'widget': Textarea(attrs={'rows': 2})},
     }
 
+    # Override the "view on site" button to use local /documentation/ instead of docs.code.org path
+    def view_on_site(self, obj):
+        return '/documentation%s' % obj.get_absolute_url()
```

Comment:
why doesn't `get_absolute_url` include this path?

Reply:
Because the path that gets used when things are actually published doesn't include documentation. This is because we're trying to publish both docs.code.org and curriculum.code.org from the same same place (docs.code.org on s3 lives at curriculum.code.org/documentation)

Reply:
Oooh, that's unfortunate. Makes sense, though